src/paho/mqtt/subscribe.py

```diff
@@ -19,53 +19,53 @@
 you to pass a callback for processing of messages.
 """
 
+import ssl
```

Comment:
I removed `import ssl` from `publish.py` and forgot to remove from `subscribe.py`.

Now it is really done!